charts/spire/README.md

```diff
@@ -30,6 +31,7 @@ spec:
         - --service-account-signing-key-file=/run/config/pki/sa.key
 ```
 
+
```

Comment:
Seems you have an issue with line endings on Windows using helm-docs.

Reply:
These unusual whitespace changes were made by helm_docs.sh